src/main/java/com/onlinebanking/backend/persistent/domain/Role.java

```diff
@@ -27,6 +29,9 @@
     private int id;
     private String name;
 
+    @OneToMany

```

Comment:
We will not retrieve users from roles so I am sure we can do without the relationship in this direction. We can remove this one.

Reply:
Okay but by database design this would be correct? We're creating userrole entity because we have a many-to-many mapping

Reply:
Yes. This is correct and it's the bi-directional design. 

We will only go one way from User to roles and we will establish the bi directional in the user_role and user not the user_role and role.